app/src/androidTest/java/br/com/catbag/gifreduxsample/ui/giflist/GifListActivityTest.java

```diff
@@ -70,12 +74,19 @@ public GifListActivityTest() {
     @Override
     public void setup() {
         super.setup();
-        GifListActionCreator.getInstance().setDataManager(mock(DataManager.class));
+        mHelper.getFluxxan().getDispatcher().unregisterMiddleware(RestMiddleware.class);
+        Middleware restMiddleware = new RestMiddleware(getContext(), mock(DataManager.class),
+                new FileDownloader());
+        mHelper.getFluxxan().getDispatcher().unregisterMiddleware(RestMiddleware.class);
```

Comment:
2 lines with this code...`mHelper.getFluxxan().getDispatcher().unregisterMiddleware(RestMiddleware.class);`